apps/admin_api/lib/admin_api/v1/error_handler.ex

```diff
@@ -130,10 +131,37 @@ defmodule AdminAPI.V1.ErrorHandler do
 
     respond(conn, code, description, messages)
   end
+
   def handle_error(conn, :invalid_parameter, description) do
     respond(conn, @errors.invalid_parameter.code, description)
   end
 
+  @doc """
+  Handles response of insufficient funds error.
+  Duplicate from eWalletAPI ErrorHandler, will be refactored in error merge.
+  """
+  def handle_error(conn, :insufficient_funds, d) do
+    handle_error(conn, "transaction:insufficient_funds", d)
+  end
+  def handle_error(conn, "transaction:insufficient_funds", %{
+    "address" => address,
+    "current_amount" => current_amount,
+    "amount_to_debit" => amount_to_debit,
+    "friendly_id" => friendly_id
+  }) do
+    minted_token = MintedToken.get(friendly_id)
+    current_amount = :erlang.float_to_binary(current_amount / minted_token.subunit_to_unit,
```

Comment:
Let's create or use a `Money` module (as a separate task)?

Reply:
Okay, we'll do it in a future task!